### PR TITLE
[BE][Easy] enable postponed annotations in `tools`

### PR DESCRIPTION
**Pull request by XuehaiPan**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #129420
* #129419
* #129376
* __->__ #129375

cc @ezyang @malfet @xuzhao9 @gramster @albanD @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang

### CODE REVIEW

**Comment by Skylion007** [comment]
See https://github.com/pytorch/pytorch/issues/117449

File: tools/alerts/create_alerts.py [hunk 1]
```diff
@@ -1,16 +1,19 @@
 #!/usr/bin/env python3
 
+from __future__ import annotations
```
Comment (malfet) [inline tools/alerts/create_alerts.py:3]: Do you mind filing an issue to get rid of those `from __future__ import annotations` as we are about to deprecated Python-3.8 soon and 3.9 already supports those

**Review by malfet — APPROVED** [review]
LGTM (and feel free to suppress BC-linter errors, as they are false positives)

**Pull request closed without merging**